swiftspec/core.py

```diff
@@ -142,18 +142,34 @@ def get_tokens_from_env(self):
         token = os.environ.get("OS_AUTH_TOKEN")
         url = os.environ.get("OS_STORAGE_URL")
         if token and url:
-            return [{"token": token, "url": url}]
+            return [{"X-Auth-Token": token, "url": url}]
```

Comment:
This change seems to be unnecessary for this PR and potentially breaks existing API (namely, users who pass in `auth` to the filesystems constructor, instead of via environment variables would have to change the call from `token` to `X-Auth-Token` as well). We should avoid unnecessary breaking changes.